examples/basic_mysql_connection/index.js

```diff
@@ -13,8 +13,23 @@ var connection = mysql.createConnection({
 
 connection.connect();
 
-connection.query('SELECT * FROM todos', function (error, results, fields) {
-    console.log(results);
-});
+let query=[	'select count(*) as items from todos',
+			'select * from todos where done',
+			'select * from todos where !done',
+			'select * from todos order by due DESC',
+			'select * from todos where due=(select max(due) from todos)',
```

Comment:
You used a subquery here, similar to Aws and Amir - I'll copy for you what I said to them:

<hr>

This is an interesting use of a subquery, and not necessarily how I would choose to write this. The question isn't super clear, so I apologise for that.

However, subqueries can be dramatically slower than other forms of the same query. Can you suggest another way this could be written which doesn't require a subquery?

For an example of why a subquery would be slower, imagine that this table has a billion rows. The subquery gets executed first, which will involve reading all one billion rows to find the max. Then, the main query will execute, which involves scanning all one billion rows again to find the match.

And, beyond that, this doesn't actually answer the question "Get the single most recently added todo item". Instead, it finds the one that's due the furthest in the future (which is not necessarily the same thing) 😛